addons/mass_mailing/static/src/js/mass_mailing_snippets.js

```diff
@@ -95,36 +95,17 @@ options.registry.mass_mailing_sizing_x = options.Class.extend({
 
 // Adding compatibility for the outlook compliance of mailings.
 // Commit of such compatibility : a14f89c8663c9cafecb1cc26918055e023ecbe42
-options.registry.BackgroundImage = options.registry.BackgroundImage.extend({
+options.registry.BackgroundImageMassMailing = options.registry.BackgroundImage.extend({
     start: function () {
         this._super();
-        if (this.snippets && this.snippets.split('.')[0] === "mass_mailing") {
-            var $table_target = this.$target.find('table:first');
-            if ($table_target.length) {
-                this.$target = $table_target;
-            }
+        var $table_target = this.$target.find('table:first');
```

Comment:
If you touch a line with a `var`, use const or let instead. Also use camelCase instead of skake_case